src/mlpack/core/dists/regression_distribution.cpp

```diff
@@ -65,6 +66,20 @@ double RegressionDistribution::Probability(const arma::vec& observation) const
   return err.Probability(observation(0)-fitted.t());
 }
 
+/**
+ * Evaluate probability density function of given observation.
+ *
+ * @param x List of observations.
+ * @param probabilities Output probabilities for each input observation.
+ */
+void RegressionDistribution::Probability(const arma::mat& x,
```

Comment:
Do you think instead of ``x`` we could go with ``observations``, would be more descriptive. Let me know what you think.